apps/common-app/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "yarn patch-package",
-    "lint": "eslint --max-warnings=0 .",
+    "lint": "eslint .",
```

Comment:
Why was `--max-warnings=0` removed?

Reply:
+1
I had assumed that the default value was `0` previously, but based on the [documentation](https://eslint.org/docs/latest/use/command-line-interface#--max-warnings), it appears that we require this flag.

Reply:
Using `--max-warnings=0` makes the tests fail if there is any warning. It was removed as we probably want CI to pass despite the warnings.

>Normally, if ESLint runs and finds no errors (only warnings), it exits with a success exit status. However, if --max-warnings is specified and the total warning count is greater than the specified threshold, ESLint exits with an error status.

After reading the attached documentation I still think we should either omit this flag or provide `--max-warnings=-1`...
Or we can have `--max-warnings="current number of warnings"` to make sure no new warnings are generated.

Reply:
so, why did it work before with this flag?

Reply:
>so, why did it work before with this flag?
Probably some of our warnings are generated by rules that weren't enabled before. The broken warnings are `@typescript-eslint/no-explicit-any` and `@typescript-eslint/no-var-requires`. We use var requires a lot for importing assets and most of explicit any are in runtime tests (for example because we don't export type for EntryExitAnimation).

In the future we can config out rule to allow import of tests and assets: https://typescript-eslint.io/rules/no-var-requires/ and fix and ban `@typescript-eslint/no-explicit-any` outside Runtime Tests. However this would be beyond the scope of this PR.

Also if certain rule should stop a PR from being merged then I would recommend marking it as error.